[2024-07-28T00:34:30.319+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-28T00:34:30.335+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: first_dag.second_task scheduled__2024-07-27T00:00:00+00:00 [queued]>
[2024-07-28T00:34:30.341+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: first_dag.second_task scheduled__2024-07-27T00:00:00+00:00 [queued]>
[2024-07-28T00:34:30.342+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2024-07-28T00:34:30.348+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): second_task> on 2024-07-27 00:00:00+00:00
[2024-07-28T00:34:30.357+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62 DeprecationWarning: This process (pid=16652) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-07-28T00:34:30.354+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'first_dag', 'second_task', 'scheduled__2024-07-27T00:00:00+00:00', '--job-id', '10', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmp3oc6ubg1']
[2024-07-28T00:34:30.357+0000] {standard_task_runner.py:64} INFO - Started process 16746 to run task
[2024-07-28T00:34:30.357+0000] {standard_task_runner.py:91} INFO - Job 10: Subtask second_task
[2024-07-28T00:34:30.391+0000] {task_command.py:426} INFO - Running <TaskInstance: first_dag.second_task scheduled__2024-07-27T00:00:00+00:00 [running]> on host 59541af12559
[2024-07-28T00:34:30.455+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='student' AIRFLOW_CTX_DAG_ID='first_dag' AIRFLOW_CTX_TASK_ID='second_task' AIRFLOW_CTX_EXECUTION_DATE='2024-07-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-07-27T00:00:00+00:00'
[2024-07-28T00:34:30.462+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-28T00:34:30.463+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-07-28T00:34:30.464+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hello world, this is 2nd task']
[2024-07-28T00:34:30.473+0000] {subprocess.py:86} INFO - Output:
[2024-07-28T00:34:30.476+0000] {subprocess.py:93} INFO - hello world, this is 2nd task
[2024-07-28T00:34:30.476+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-07-28T00:34:30.477+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-28T00:34:30.500+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=first_dag, task_id=second_task, run_id=scheduled__2024-07-27T00:00:00+00:00, execution_date=20240727T000000, start_date=20240728T003430, end_date=20240728T003430
[2024-07-28T00:34:30.531+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-07-28T00:34:30.542+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-28T00:34:30.543+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
